### ISSUE TITLE
Bug: Root level $ref not being parsed

### ISSUE THREAD

**Issue by willfarrell** [BigstickCarpet/json-schema-ref-parser#16]
```javascript
var input = { "$ref":"http://192.168.99.100/schema/sandbox#/definitions/item" };
$RefParser.dereference(input, function(err, schema) {
		if (err) {
			console.error(err);
			return done = true;
		}
		 console.log(schema); 
});
// console:
// { '$ref': 'http://192.168.99.100/schema/sandbox#/definitions/item' }
```

**Comment by BigstickCarpet** [comment]
This should be fixed now, as of [version 3.0.0](https://github.com/BigstickCarpet/json-schema-ref-parser/blob/master/CHANGELOG.md#v300-2016-04-03).

**Comment by willfarrell** [comment]
Thanks!
